kqueen/engines/gce.py

```diff
@@ -170,6 +174,16 @@ def deprovision(self, **kwargs):
         return True, None
 
     def resize(self, node_count, **kwargs):
+
+        if node_count < 2 and \
+           self.cluster_config['cluster']['networkPolicy']['enabled'] is True:
```

Comment:
cluster may not have 'networkPolicy' so you should check it with _get_ before trying to get enabled

Reply:
it should be anyway, coz i defined network policy in config on __init__ method